Packs/PrismaCloudCompute/Scripts/TagIndicatorButton/TagIndicatorButton.py

```diff
@@ -0,0 +1,5 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+args = demisto.args()
+res = demisto.executeCommand("setIndicators", {"indicatorsValues": args.get("Indicators"), "tags": args.get("Tags")})
```

Comment:
please do the following in all the places you used **demisto.executeCommand**
```suggestion
res = demisto.executeCommand("setIndicators", {"indicatorsValues": args.get("Indicators"), "tags": args.get("Tags")})
if is_error(res):
    raise DemistoException(f'Failed to set indicators: {str(get_error(res))}')

```